crm_phonecall/wizard/crm_phonecall_to_phonecall.py

```diff
@@ -56,16 +56,17 @@ def action_schedule(self):
         phonecall_obj = self.env['crm.phonecall']
         phonecalls = phonecall_obj.browse(self._context.get('active_ids', []))
         for this in self:
-            phonecalls.schedule_another_phonecall(
+            new_phonecalls_dict = phonecalls.schedule_another_phonecall(
```

Comment:
If I'm not wrong, you are here generating phone calls in an exponential fashion:

1. `phonecalls` contains N recordsets.
2. You iterate over `phonecalls` and call `schedule_another_phonecall()` on each of those with the current values, thus creating N * N phonecalls.
2. You only store the last call's result into `new_phonecalls_dict`, making it seem like only N phonecalls were created.

So either:

- Call `this.schedule_another_phonecall`
- Call only once (without `for` loop) `phonecalls.schedule_another_phonecall`
- I'm completely wrong and I don't understand what you're doing here